docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
For reference, other ideas are discussed in #2144.

I'm surprised some flavor of `stop` wasn't included in this proposal. Was that intentional?